build/Dockerfile

```diff
@@ -8,13 +8,20 @@ ENV DEBIAN_FRONTEND=noninteractive DEBIAN_PRIORITY=critical
 RUN yes | unminimize > /dev/null 2>&1
 
 # curl and jq are prerequisites for the install-kernel-package.sh script
-RUN apt-get update -qq && apt-get -qq -y full-upgrade > /dev/null 2>&1 && apt-get install -y -qq jq curl
+RUN apt-get update -qq > /dev/null 2>&1 && apt-get -qq -y full-upgrade > /dev/null 2>&1 && apt-get install -y -qq jq curl > /dev/null 2>&1
 
 # Used to determine the latest kernel version in the script
 ARG NETBOOT_IP
 COPY kernel/install-kernel-packages.sh install-kernel-packages.sh
 RUN ./install-kernel-packages.sh $NETBOOT_IP
 
+# Install bluetooth drivers
+RUN bash -c 'apt-get download linux-firmware && \
+    dpkg -x *.deb /tmp/linux-firmware/ && \
+    cp -r /tmp/linux-firmware/lib/firmware/iwlwifi-* /lib/firmware/ && \
```

Comment:
```suggestion
    cp -r /tmp/linux-firmware/lib/firmware/iwlwifi-* /lib/firmware/ && \
    mkdir /lib/firmware/intel/ && \ 
    cp /tmp/linux-firmware/lib/firmware/intel/ibt-0041-0041* /lib/firmware/intel/ && \
```

That should make it work with ax210. I think.
As mentionned in our direct chat, I saw the following kernel messages:

```sh
  19.476916] Bluetooth: hci0: Device revision is 0
[   19.476921] Bluetooth: hci0: Secure boot is enabled
[   19.476922] Bluetooth: hci0: OTP lock is enabled
[   19.476923] Bluetooth: hci0: API lock is enabled
[   19.476923] Bluetooth: hci0: Debug lock is disabled
[   19.476924] Bluetooth: hci0: Minimum firmware build 1 week 10 2014
[   19.476925] Bluetooth: hci0: Bootloader timestamp 2019.40 buildtype 1 build 38
[   19.479715] Bluetooth: hci0: Failed to load Intel firmware file intel/ibt-0041-0041.sfi (-2)
[   19.480895] Bluetooth: hci0: Failed to read MSFT supported features (-56)
[   19.483609] Intel(R) Wireless WiFi driver for Linux
[   19.614921] iwlwifi 0000:a9:00.0: api flags index 2 larger than supported by driver
[   19.614939] iwlwifi 0000:a9:00.0: TLV_FW_FSEQ_VERSION: FSEQ Version: 0.0.2.36
[   19.639031] iwlwifi 0000:a9:00.0: loaded firmware version 72.a764baac.0 ty-a0-gf-a0-72.ucode op_mode iwlmvm
```

On this system, I copied the 2 files above into `/lib/firmware/intel` and reinitialized the hci0 device:

```sh
# hciconfig hci0 down
# rmmod btusb
# modprobe btusb
# hciconfig hci0 up
```

... then checked it with `dmesg` again. After these steps I successfully connected my Bluetooth mouse (MX Master 3)

But: I think with every new iteration, we can start this debugging process again, if Bluetooth doesn't work.